### ISSUE TITLE
[Paper UI] Configuration input ignores "pattern" attribute

### ISSUE THREAD

**Issue by kubawolanin** [eclipse/smarthome#2862]
In `thing-types.xml` I'm able to define my parameters for the binding's Thing.
According to [the documentation](https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-configuration-descriptions), I should be able to pass `pattern` property, which allows me to define validation Regular Expression.

In Paper UI, however, I'm unable to do it.

Let's say I have a parameter receiving geo coordinates:
```xml
            <parameter name="location" type="text" required="true" 
            pattern="^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?)[,|;]\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$">
                <label>Location</label>
                <description>Your geo coordinates separated with semicolon (e.g. "37.8;-122.4").</description>
            </parameter>
```

When everything is done, Paper UI renders HTML5 `<input>` control without the `pattern` property.
![pattern](https://cloud.githubusercontent.com/assets/2270505/22082792/80ac585e-ddc9-11e6-9d18-6d114227a164.PNG)

Thank you,
Kuba

**Comment by kaikreuzer** [comment]
Is this a duplicate of https://github.com/eclipse/smarthome/issues/2556?
If so, could you check if https://github.com/eclipse/smarthome/pull/2812 meets your requirements?

**Comment by kubawolanin** [comment]
Absolutely! Seems I overlooked this one.
Thank you. 
Closing as it's a duplicate of #2556 being resolved by #2812
